src/app/component/CategoryFilterOverlay/CategoryFilterOverlay.component.js

```diff
@@ -128,9 +129,12 @@ export default class CategoryFilterOverlay extends PureComponent {
         const {
             isInfoLoading,
             availableFilters,
+            totalPages,
             onVisible
         } = this.props;
 
+        if (totalPages === 0) return <Overlay mix={ { block: 'CategoryFilterOverlay' } } id="category-filter" />;
```

Comment:
Please, stop using single line returns, format the code properly:

```
if (totalPages === 0) {
    return (
        <Overlay
            mix={ { block: 'CategoryFilterOverlay' } }
            id="category-filter"
         />
    );
}
```

Prefer const for popup id.